Source/Public/Get-AZDOPSPool.ps1

```diff
@@ -0,0 +1,51 @@
+function Get-AZDOPSPool {
+    [CmdletBinding(DefaultParameterSetName = 'All')]
+    param (
+        [Parameter(ParameterSetName = 'PoolId')]
+        [Parameter(ParameterSetName = 'PoolName')]
+        [Parameter(ParameterSetName = 'All')]
+        [string]$Organization,
```

Comment:
If no organization is given the URL will not work. Please change this to mandatory, or use the $org['organization'] variable in the URL as that one will always be populated.

Reply:
Would probably help if i read the code right though.. sorry.. :/